spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/ConfigServerConfiguration.java

```diff
@@ -42,7 +42,6 @@ public SpringApplicationEnvironmentRepository EnvironmentRepository() {
 	}
 
 	@Configuration
-	@Profile("!native")
```

Comment:
I wasnt sure about that. My understanding is that if we keep a "git" Profile it must be activated explicitly. 
This way the default is Git, but can be overriden (to "native" or "subversion").